src/core/Akka.Streams/Stage/Stage.cs

```diff
@@ -122,7 +122,7 @@ public abstract class PushStage<TIn, TOut> : PushPullStage<TIn, TOut>
     /// @see <see cref="PushPullStage{TIn,TOut}"/>
     /// </summary>
     [Obsolete("Please use GraphStage instead.")]
-    public abstract class DetachedStage<TIn, TOut> : AbstractStage<TIn, TOut, IUpstreamDirective, IDownstreamDirective, IDetachedContext<TOut>, ILifecycleContext>
+    public abstract class DetachedStage<TIn, TOut> : AbstractStage<TIn, TOut, IUpstreamDirective, IDownstreamDirective, IDetachedContext<TOut>>
```

Comment:
I'm not sure for this stage but in general we are still using these obsolete stages internally and will move to GraphStage over time. Once all stages are using GraphStage we can remove them.